src/styles/blocks/month.scss

```
@@ -0,0 +1,46 @@
+.month {
+  padding: 0 $month-padding;
+  width: getMonthWidth($month-padding);
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+
+  &__day {
+    @include square($day-width);
+    background-color: #eee;
+    border: 1px solid black;
+    position: relative;
+    cursor: pointer;
+
+    @for $day from 1 through $days-in-month {
+      &--#{$day}::before {
+        position: absolute;
+        content: "#{$day}";
+        top: 50%;
+        left: 50%;
+        transform: translate(-50%, -50%);
+        color: black;
+        font-size: 30px;
+        font-family: Arial, Helvetica, sans-serif;
+      }
+    }
+
+    &:hover {
+      background-color: pink;
+      transform: translateY(-20px);
+      transition: all 500ms;
```

https://user-images.githubusercontent.com/38065883/122228801-e242bf80-cec0-11eb-8cde-6a6723291003.mov

transition щас работает только в одну сторону, добавляе его на сам класс, а не на hover, потому что это правило получается пропадает, когда ты убираешь мышку с элемента
И указывай какое именно свойство ты анимируешь
```suggestion
      transition: transform 500ms;
```